### ISSUE TITLE
The problem with bots in the API

### ISSUE THREAD

**Issue by Pokimoshka** [Next21Team/NextClientServerApi#3]
Hello, will there be some kind of bot exception in ncl_is_next_client(id)? All because the API returns NULL

```
if (utils::PlayerByIndex(params[x]) == NULL) { 
	MF_LogError(amx, AMX_ERR_NATIVE, "%s: disconnected player %i [%s]", __FUNCTION__, params[x], #x); 
	return FALSE; 
}
```

**Comment by Polarhigh** [comment]
@Xelson What do you think?

**Comment by Xelson** [comment]
But `ncl_is_next_client` should return `false` for bots, what's wrong?

**Comment by Pokimoshka** [comment]
@Xelson @Polarhigh Without checking for is_user_bot, errors immediately appear in the logs
`[nclapi] ncl_is_next_client: disconnected player 1 [arg_index]`

**Comment by Xelson** [comment]
Fixed in https://github.com/Next21Team/NextClientServerApi/commit/f346ab02cedfa85c95b44ebe4d9bf75a51c8ab25

**Comment by Pokimoshka** [comment]
@Nord1cWarr1or  PodBot/YaPb

**Comment by Polarhigh** [comment]
Fixed in [v1.3.1](https://github.com/Next21Team/NextClientServerApi/releases/tag/v1.3.1)
